config/rbac/kustomization.yaml

```diff
@@ -9,6 +9,8 @@ resources:
   - role.yaml
   - role_binding.yaml
   - service_account.yaml
+  - networkplugin_syncer_cluster_role.yaml
+  - networkplugin_syncer_cluster_role_binding.yaml
```

Comment:
What is this file for?

Reply:
@SteveMattar you’ll probably be able to explain this better than me...

Reply:
I put these in here assuming they're needed.

Reply:
@tpantelis its a list of rbac resources which `kustomize` is using to build the final bundle for OLM.
It's needed for operatorhub.io basically.

Reply:
I believe that if you run `kubectl kustomize .`  in this directory, you will get the bundle.yaml out (which is used to install the operator just from a yaml)

Reply:
but only the rbacs part

Reply:
You can run `make bundle`